src/plugins/ui/edition.ts

```diff
@@ -198,7 +202,7 @@ export class EditionPlugin extends UIPlugin {
         const sheetIdExists = !!this.getters.tryGetSheet(this.sheetId);
         if (!sheetIdExists && this.mode !== "inactive") {
           this.sheetId = this.getters.getActiveSheetId();
-          this.cancelEdition();
+          this.cancelEditionAndActivateSheet();
```

Comment:
this line  was shady already.  we first set this.sheetId, which means the activateSheet part will never occur...

Seems louche to me to set this.sheetId even though you're not editing anything? I think that cancelling should probably reset the value of this.sheetId as well. That being said, looks more like a refactoring than a fix. I wll add this in the llst of the stuff we need to address when refactoring the composer. The whole internal state is a mess.